packages/service/src/modules/auth/login-service.ts

```diff
@@ -26,13 +31,20 @@ export class LoginService {
       user = await createUserService.create({
         name: userInfo.name,
         email: userInfo.email,
+        slackHandle,
+        image,
       });
+    } else {
+      // Update user's profile image
+      logger.debug(`Updating ${user.email}'s profile picture.`);
+      if (!user.slackHandle) user.slackHandle = slackHandle; // If user.slackHandle is null, updating below will fail
+      await new UpdateUserService(this.context).update({ ...user, image });
     }
 
     let scopes = ['cru:achievements', 'cru:awards', 'r:users', 'ru:profile', 'r:highscore', 'r:feature'];
 
     // Administrator rights for certain users
-    const isAdmin = userInfo.email === 'niclas.lindstedt@etimo.se';
+    const isAdmin = userInfo.email === 'axel.elmarsson@etimo.se';
```

Comment:
;)

Reply:
Kan du lägga in mig här också så att jag kan fortsätta vara admin?

Reply:
Juste, skulle inte commita det där egentligen :D Fixar